contracts/KeyInfrastructure.sol

```diff
@@ -46,7 +63,29 @@ contract KeyInfrastructure {
         return _intermediate;
     }
 
-    function getIssuerKey() public view returns (address) {
-        return _issuer;
+    function getActiveIssuers() public view returns (address[] memory) {
+        return _issuers;
+    }
+
+    function isActiveIssuer(address addr) public view returns (bool) {
+        return _isActiveIssuer[addr];
+    }
+
+    function addToIssuers(address addr) internal {
```

Comment:
```suggestion
    function _activateIssuer(address addr) internal {
```

Reply:
This one I would disagree with, the `activate` terminology is kind of reserved mainly for detailing the high level intent of an external/public function, whereas internal functions I see as lower level which are more dev-focused and should be more descriptive about what they actually do. It's the difference between e.g. `extend` and `addToExtensionList`.

But yes, underscore required!

Reply:
Good point, make sense. 
If we want to be totally descriptive about what it does, then what about `_addToActiveIssuers`? 🙂

Reply:
Ah yes! Perfect :) I feel that we are slowly sharpening our sword 🗡️